options/options.js

```diff
@@ -0,0 +1,48 @@
+// Saves options to chrome.storage.sync.
+function save_options() {
+  var MB = document.getElementById('MB').value;
+  
+  chrome.storage.local.set({
+    MB: MB
+  }, function() {
+    // Update status to let user know options were saved.
+    var status = document.getElementById('status');
+    status.textContent = 'Options saved.';
+    setTimeout(function() {
+      status.textContent = '';
+    }, 1050);
+  });
+}
+
+// Restores option state using the preferences stored in chrome.storage.
+function restore_options() {
+    chrome.storage.local.get('MB', function(items) {
+        var MB;
+        if(items.MB !== undefined) {
+            MB = items.MB;
```

Comment:
If I remember correctly from the doc we also have to align the memory value to powers of 2 (see: https://groups.google.com/d/msg/emscripten-discuss/FJ9Y4qNg2to/6jqeFmZUeTAJ). Either round up to the nearest power of two if the user tries to set anything that isn't one or change the textfield to a dropdown?